Unknown Series in kafka-tls.yaml

Deploying kafka-tls.yaml fails because of a series declaration (focal) that is unsupported at this time. 
```
Run tree -fi releases/ | grep .yaml | grep -v k8s |  while read BUNDLE
integration create: /home/runner/work/kafka-bundle/kafka-bundle/.tox/integration
integration installdeps: pytest, juju, kazoo, pure-sasl, tenacity, pytest-operator, -r/home/runner/work/kafka-bundle/kafka-bundle/requirements.txt
integration installed: asttokens==2.0.8,attrs==22.[1](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:1).0,backcall==0.2.0,bcrypt==4.0.0,cachetools==5.2.0,certifi==2022.[9](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:10).14,cffi==1.15.1,charset-normalizer==2.1.1,cryptography==38.0.1,decorator==5.1.1,executing==1.0.0,google-auth==2.11.1,idna==3.4,iniconfig==1.1.1,ipdb==0.13.9,ipython==8.5.0,jedi==0.18.1,Jinja2==3.1.2,juju==3.0.1,jujubundlelib==0.5.7,kazoo==2.8.0,kubernetes==24.2.0,macaroonbakery==1.3.1,MarkupSafe==2.1.1,matplotlib-inline==0.1.6,mypy-extensions==0.4.3,oauthlib==3.2.1,ops==1.5.2,packaging==21.3,paramiko==2.11.0,parso==0.8.3,pexpect==4.8.0,pickleshare==0.7.5,pluggy==1.0.0,prompt-toolkit==3.0.31,protobuf==3.20.2,ptyprocess==0.7.0,pure-eval==0.2.2,pure-sasl==0.6.2,py==1.11.0,pyasn1==0.4.8,pyasn1-modules==0.2.8,pycparser==2.21,Pygments==2.13.0,pymacaroons==0.13.0,PyNaCl==1.5.0,pyparsing==3.0.9,pyRFC3339==1.1,pytest==7.1.3,pytest-asyncio==0.19.0,pytest-operator==0.22.0,python-dateutil==2.8.2,pytz==2022.2.1,PyYAML==6.0,requests==2.28.1,requests-oauthlib==1.3.1,rsa==4.9,six==1.16.0,stack-data==0.5.0,tenacity==8.1.0,theblues==0.5.2,toml==0.[10](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:11).2,tomli==2.0.1,toposort==1.7,traitlets==5.4.0,typing-inspect==0.8.0,typing_extensions==4.3.0,urllib3==1.26.12,wcwidth==0.2.5,websocket-client==1.4.1,websockets==7.0
integration run-test-pre: PYTHONHASHSEED='565670733'
integration run-test: commands[0] | pytest -vv --no-header --tb native --log-cli-level=INFO -s --bundle releases/latest/machine/kafka-tls.yaml
============================= test session starts ==============================
collecting ... collected 4 items

tests/integration/test_bundle.py::test_deploy_bundle_active[releases/latest/machine/kafka-tls.yaml] 
-------------------------------- live log setup --------------------------------
INFO     pytest_operator.plugin:plugin.py:625 Adding model github-pr-1519b:test-bundle-7tmo on cloud localhost
-------------------------------- live log call ---------------------------------
INFO     pytest_operator.plugin:plugin.py:[11](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:12)37 Deploying (and possibly building) bundle using juju-bundle command:'juju-bundle deploy --bundle releases/latest/machine/kafka-tls.yaml --'
FAILED
tests/integration/test_bundle.py::test_deploy_app_charm_relate[releases/latest/machine/kafka-tls.yaml] XFAIL (aborted)
tests/integration/test_bundle.py::test_run_action_produce XFAIL (aborted)
tests/integration/test_bundle.py::test_run_action_consume XFAIL (aborted)
------------------------------ live log teardown -------------------------------
INFO     pytest_operator.plugin:plugin.py:768 Model status:

Model             Controller       Cloud/Region         Version  SLA          Timestamp
test-bundle-7tmo  github-pr-1519b  localhost/localhost  2.9.34   unsupported  15:28:48Z



INFO     pytest_operator.plugin:plugin.py:774 Juju error logs:


INFO     pytest_operator.plugin:plugin.py:790 juju-crashdump finished [0]
INFO     pytest_operator.plugin:plugin.py:861 Resetting model test-bundle-7tmo...
INFO     pytest_operator.plugin:plugin.py:866 Not waiting on reset to complete.
INFO     pytest_operator.plugin:plugin.py:839 Forgetting main...


=================================== FAILURES ===================================
______ test_deploy_bundle_active[releases/latest/machine/kafka-tls.yaml] _______
Traceback (most recent call last):
  File "/home/runner/work/kafka-bundle/kafka-bundle/tests/integration/test_bundle.py", line 45, in test_deploy_bundle_active
    charm = await ops_test.deploy_bundle(bundle=bundle, build=False)
  File "/home/runner/work/kafka-bundle/kafka-bundle/.tox/integration/lib/python3.8/site-packages/pytest_operator/plugin.py", line 1141, in deploy_bundle
    await self.run(*cmd, check=True)
  File "/home/runner/work/kafka-bundle/kafka-bundle/.tox/integration/lib/python3.8/site-packages/pytest_operator/plugin.py", line 595, in run
    raise AssertionError(
AssertionError: Command ['juju-bundle', 'deploy', '--bundle', 'releases/latest/machine/kafka-tls.yaml', '--'] failed (1): Error: YAML Error: series: unknown variant `focal`, expected one of `kubernetes`, `oneiric`, `precise`, `quantal`, `raring`, `saucy`, `trusty`, `utopic`, `vivid`, `wily`, `xenial`, `yakkety`, `zesty`, `artful`, `bionic`, `cosmic`, `disco`, `eoan`, `win20[12](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:13)hvr2`, `win2012hv`, `win2012r2`, `win2012`, `win7`, `win8`, `win81`, `win10`, `win2016`, `win2016hv`, `win2016nano`, `centos7` at line 30 column 9

Caused by:
    series: unknown variant `focal`, expected one of `kubernetes`, `oneiric`, `precise`, `quantal`, `raring`, `saucy`, `trusty`, `utopic`, `vivid`, `wily`, `xenial`, `yakkety`, `zesty`, `artful`, `bionic`, `cosmic`, `disco`, `eoan`, `win2012hvr2`, `win2012hv`, `win2012r2`, `win2012`, `win7`, `win8`, `win81`, `win10`, `win2016`, `win2016hv`, `win2016nano`, `centos7` at line 30 column 9
------------------------------ Captured log setup ------------------------------
INFO     pytest_operator.plugin:plugin.py:625 Adding model github-pr-1519b:test-bundle-7tmo on cloud localhost
------------------------------ Captured log call -------------------------------
INFO     pytest_operator.plugin:plugin.py:1[13](https://github.com/canonical/kafka-bundle/actions/runs/3106693376/jobs/5033894363#step:4:14)7 Deploying (and possibly building) bundle using juju-bundle command:'juju-bundle deploy --bundle releases/latest/machine/kafka-tls.yaml --'
=========================== short test summary info ============================
FAILED tests/integration/test_bundle.py::test_deploy_bundle_active[releases/latest/machine/kafka-tls.yaml]
======================== 1 failed, 3 xfailed in 10.70s =========================
ERROR: InvocationError for command /home/runner/work/kafka-bundle/kafka-bundle/.tox/integration/bin/pytest -vv --no-header --tb native --log-cli-level=INFO -s --bundle releases/latest/machine/kafka-tls.yaml (exited with code 1)
```

--- a/releases/latest/machine/kafka-tls.yaml
+++ b/releases/latest/machine/kafka-tls.yaml
@@ -2,13 +2,11 @@ applications:
   kafka:
     channel: edge
     charm: kafka
-    constraints: arch=amd64
     num_units: 3
     revision: 68
   tls-certificates-operator:
     channel: edge
     charm: tls-certificates-operator
-    constraints: arch=amd64
     num_units: 1
     options:
       ca-common-name: canonical
@@ -17,7 +15,6 @@ applications:
   zookeeper:
     channel: edge
     charm: zookeeper
-    constraints: arch=amd64
     num_units: 3
     revision: 65
 relations:
@@ -27,4 +24,3 @@ relations:
   - zookeeper:certificates
 - - tls-certificates-operator:certificates
   - kafka:certificates
-series: focal

